backend/database-collection-schemas/createAnnouncementCollection.py

```diff
@@ -105,14 +119,37 @@
     sys.exit(1)
 # Create some fake data
 data = [
-    (1637100804, "Message _8_04", "Content for Message _8_04", "robot"),
-    (1637100704, "Message _7_04", "Content for Message _7_04", "robot"),
-    (1637100604, "Message _6_04", "Content for Message _6_04", "robot"),
-    (1637100504, "Message _5_04", "Content for Message _5_04", "robot"),
-    (1637100404, "Message _4_04", "Content for Message _4_04", "robot"),
-    (1637100304, "Message _3_04", "Content for Message _3_04", "robot"),
-    (1637100204, "Message _2_04", "Content for Message _2_04", "robot"),
-    (1637100104, "Message _1_04", "Content for Message _1_04", "robot"),
+    (
+        1637100804,
+        "A new Announcement!",
+        "Today is a great day, a day of heroines and super power. While the world...",
+        "robot",
+    ),
+    (
+        1647100704,
+        "Parappas Mega Barbecue Party",
+        "Rapper Parappa schmeißt die Party des Jahrhunderts. Reichlich Nudeln, Burger und ulkige Showeinlagen ist noch untertrieben. Spitze Witze, würzige Fürze und schleimige Reime sollte man nicht verpassen lassen. Zur Kür ein Geschwür, äh. Rapper-Prepper und ein Rabbiat-Akkrobat sorgen für Spaß, privat. Keine Zeit so weit?? Ja bist du gescheit? Gefeiert wird bis's schneit. So der Eid. Komm einfach, etwas Besseres hast du nicht zu tun.",
+        "robot",
+    ),
+    (
+        1637100604,
+        "Habe meinen Verstand verloren",
+        "Kann mir irgendwer helfen suchen? Das muss am Amber Stow Square passiert sein.",
+        "robot",
+    ),
+    (
+        1647100504,
+        "Watch out to catch a Gimbleby!",
+        "It's time where the Gimbleby sprites start to awake from their sleep. Look out and if you are lucky you catch one and get granted one wish!",
+        "robot",
+    ),
+    (
+        1647100404,
+        "Extreme Shopping Experience",
+        "Be one of the first to visit Xalax Ulcon at Heyla Dultherton's store today and win a first class augmented reality tour, the ultimate shopping experience.",
+        "robot",
+    ),
+    (1637100304, "Bella Balla", "Tam tam, tschuff, aba babba du", "robot"),
```

Comment:
Maybe we can replace this with something more generic?

Reply:
Yeah^^. I wanted to save my announcements somewhere. I can remove this since it is not relevant.